src/Semantic/CLI.hs

```diff
@@ -57,13 +58,13 @@ main = do
   (options, task) <- customExecParser (prefs showHelpOnEmpty) arguments
   config <- defaultConfig options
   res <- withTelemetry config $ \ (TelemetryQueues logger statter _) ->
-    Task.runTask (Task.TaskSession config "-" (optionsLogPathsOnError options) logger statter) task
+    Task.runTask (Task.TaskSession config "-" (optionsLogPathsOnError options) logger statter) (Parse.runParse task)
```

Comment:
`runTask` doesn’t run the parser, for two reasons:

1. A lot of the time when we do `runTask`, we’re only doing it to get all the effects that the `Measured` parse carrier requires; by splitting them up, it’ll be a bit easier to start doing away with `runTask` altogether.

2. I didn’t want to deal with circular imports between the measured carrier and `Semantic.Task` to deal with the `TaskSession`, so `Semantic.Task` doesn’t import the measured parse carrier.